modules/likes.php

```diff
@@ -626,6 +626,12 @@ function jetpack_post_likes_register_rest_field() {
 				),
 			)
 		);
+
+		/**
+		 * Ensures all public internal post-types support `likes`
+		 * This feature support flag is used by the REST API and Gutenberg.
+		 */
+		add_post_type_support( $post_type, 'likes' );
```

Comment:
Would it be an option to use a prefixed name here, to avoid conflicts with other plugins that may declare support for their own `likes`?